lib/src/data/repositories/cloud_buzzer_repository.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter_door_buzzer/src/data/managers/buzzer_api_manager.dart';
+import 'package:flutter_door_buzzer/src/data/models/api_models.dart';
+import 'package:flutter_door_buzzer/src/data/repositories/buzzer_repository.dart';
+import 'package:meta/meta.dart';
+
+class CloudBuzzerRepository extends BuzzerRepository {
+  final BuzzerApiManager buzzerApiManager;
+
+  CloudBuzzerRepository({@required this.buzzerApiManager})
+      : assert(buzzerApiManager != null),
+        super();
+
+  @override
+  Future<AuthLoginResponseModel> login({
+    @required String email,
+    @required String password,
+  }) async {
+    return await buzzerApiManager.login(email: email, password: password);
```

Comment:
Do we need to pass also a `baseUrl` param?

I see a segfault:
```
I/flutter (31837): 'package:flutter_door_buzzer/src/data/managers/buzzer_api_manager.dart':

Failed assertion: line 16 pos 16: 'baseUrl != null': No base url given
```

Reply:
sorry, I'm a bit confused.

I'd like to understand the reason for the runtime error. On a second thought, it seems that it fails to load the `config.json` file (`Unhandled Exception: Unable to load asset: config.json`).

That config file is supposed to be in `/lib`, correct?